docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -201,6 +211,107 @@ Cluster charges are based on the total number of vCPUs used and how long they ha
 
 Before creating a cluster, you need to create your billing profile and add a payment method. Refer to [Manage your billing profile and payment method](../../cloud-admin/cloud-billing-profile/).
 
+## Cloud provider regions
+
+{{< tabpane text=true >}}
+
+  {{% tab header="AWS" lang="aws" %}}
+
+The following **AWS regions** are available:
+
+- Cape Town (af-south-1)
+- Hong Kong (ap-east-1)
+- Tokyo (ap-northeast-1)
+- Seoul (ap-northeast-2)
+- Osaka (ap-northeast-3)
+- Mumbai (ap-south-1)
+- Singapore (ap-southeast-1)
+- Sydney (ap-southeast-2)
+- Central (ca-central-1)
+- Frankfurt (eu-central-1)
+- Stockholm (eu-north-1)
+- Milan (eu-south-1)
+- Ireland (eu-west-1)
+- London (eu-west-2)
+- Paris (eu-west-3)
+- Bahrain (me-south-1)
+- Sao Paulo (sa-east-1)
+- N. Virginia (us-east-1)
+- Ohio (us-east-2)
+- N. California (us-west-1)*
+- Oregon (us-west-2)
+
+\* Region has 2 availability zones only
+
+  {{% /tab %}}
+
+  {{% tab header="Azure" lang="azure" %}}
+
+The following **Azure regions** are available:
+
+- New South Wales (australiaeast)
```

Comment:
This is the list of regions that we support.

New South Wales   australiaeast      🇦🇺  AU
Sao Paulo State   brazilsouth        🇧🇷  BR
Toronto           canadacentral      🇨🇦  CA
Pune              centralindia       🇮🇳  IN
Iowa              centralus          🇺🇸  US
Hong Kong         eastasia           🇭🇰  HK
Virginia          eastus             🇺🇸  US
Virginia          eastus2            🇺🇸  US
Paris             francecentral      🇫🇷  FR
Tokyo, Saitama    japaneast          🇯🇵  JP
Seoul             koreacentral       🇰🇷  KR
Ireland           northeurope        🇮🇪  IE
Norway            norwayeast         🇳🇴  NO
Johannesburg      southafricanorth   🇿🇦  ZA
Texas             southcentralus     🇺🇸  US
Zurich            switzerlandnorth   🇨🇭  CH
Dubai             uaenorth           🇦🇪  AE
London            uksouth            🇬🇧  GB
Washington        westus2            🇺🇸  US
Phoenix           westus3            🇺🇸  US

Reply:
No Frankfurt, Singapore, Netherlands